src/sentry/lang/native/plugin.py

```diff
@@ -27,7 +27,8 @@
 
 class NativeStacktraceProcessor(StacktraceProcessor):
     supported_platforms = ('cocoa', 'native')
-    supported_images = ('apple', 'symbolic')
+    # TODO(ja): Clean up all uses of image type "apple", "uuid", "id" and "name"
+    supported_images = ('apple', 'symbolic', 'elf', 'macho', 'pe')
```

Comment:
Refactoring of the native plugin and minidump extraction will land in a follow-up PR. First, the UI and API need to be able to handle the new format.